src/workerd/io/compatibility-date.capnp

```diff
@@ -732,4 +732,12 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("allow_importable_env");
   # When allowed, `import { env } from 'cloudflare:workers'` will provide access
   # to the per-request environment/bindings.
+
+  assetsSecFetchModeNavigateHeaderPrefersAssetServing @79 :Bool
+      $compatEnableFlag("assets_navigation_prefers_asset_serving")
+      $compatDisableFlag("assets_navigation_has_no_effect")
+      $compatEnableDate("2025-04-01")
```

Comment:
This should probably be added when the experimental flag is removed and not before.

Reply:
Sorry, copy-pasta from another flag. I think we're good for this to land without `$experimental` :)